Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM node:6-stretch
+LABEL maintainer="will.price94@gmail.com"
+LABEL version="0.0.1"
+RUN apt-get update -qq && \
+    apt-get install -y python
+# Prevent npm from spamming
+RUN npm config set loglevel=warn && \
+    npm config set progress=false
+WORKDIR /app/
+COPY package.json package-lock.json /app/
+RUN npm install
+COPY . /app/
+RUN npm run build
+
+WORKDIR /app
+VOLUME /app/db
+EXPOSE 3000
+ENV VIMFLOWY_PASSWORD=vimflowy123
+ENTRYPOINT npm start -- \
+    --prod \
+    --host 0.0.0.0 \
+    --port 3000 \
+    --db sqlite \
+    --dbfolder /app/db \
+    --password $VIMFLOWY_PASSWORD
```

Comment:
totally optional since it may be complicated, just curious:  is there a way to, during the build, check that this command will work at runtime?  i.e. to hit the container on 3000 and verify it gets a response

Reply:
Not during build, but it's common to do integration tests on the build container, along the lines of...

* spin up container
* run integration test suite against container (e.g. using selenium)
* kill container
* finally push to dockerhub

Reply:
This is a non-trivial amount of work. Are there any integration tests we can leverage? I think it'd be a good idea to add this, but not as part of this pull req.

Reply:
yeah makes sense